### PR TITLE
Notes after second meeting

### PR DESCRIPTION
**Pull request by MayaGans**
- Answering some questions
- Cleaning up and removing HTML files

### CODE REVIEW

File: QandA/QandA.Rmd [hunk 1]
```diff
@@ -27,61 +27,243 @@ htmltools::img(src = knitr::image_uri("r4ds.png"),
...
+x <- c(1,2,3)
+```
+
+x exists in memory.
+
+```{r, eval=FALSE}
+z <- crazyfunction(x) 
+```
+
+`z` now points at `x`, and `eh` still doesn't exist (except metaphorically in Canada). `eh` was created and exists WHILE `crazyfunction()` was being run, but doesn't get saved to the global environment, so after the function is run you can't see its memory reference.
```
Comment (tanho63) [inline QandA/QandA.Rmd:56]: Lol you kept this exactly 😂

File: QandA/QandA.Rmd [hunk 2]
```diff
@@ -27,61 +27,243 @@ htmltools::img(src = knitr::image_uri("r4ds.png"),
...
+# subtract the median of each column from each value in the column
 for (i in seq_along(medians)) {
   x[[i]] <- x[[i]] - medians[[i]]
 }
 ```
 :::
 
-XXX
\ No newline at end of file
+If we run the following chunk, `dubug`ing our loop
```
Comment (tanho63) [inline QandA/QandA.Rmd:113]: Typo?

File: QandA/QandA.Rmd (hunk 1 above)
Comment (MayaGans) [reply QandA/QandA.Rmd:56]: I mean I loved it - hope that's okay! Feel free to edit...

**Comment by MayaGans** [comment]
Good catch on the typo, fixed I think!

**Pull request merged**